### DISCUSSION TITLE
High CPU Usage on 1.16.1 - logparser

### DISCUSSION THREAD

**Discussion by lmfrazao** [modoboa/modoboa#2031] [Q&A] [answered]
# Impacted versions

* Modoboa: 1.16.1
* installer used: No , upgrade
* Webserver: Nginx

# Steps to reproduce

Upgraded from 1.15

# Current behavior

CPU high load since last update, any solution for this ? disable logparser cronjob ?
5     *       *       *       *       root    $PYTHON $INSTANCE/manage.py logparser &> /dev/null

# Expected behavior

# Video/Screenshot link (optional)

![Screenshot_3](https://user-images.githubusercontent.com/33859834/95523715-44a8ff00-09c7-11eb-86d2-fcce0951afbb.jpg)

**Comment by MrGeneration** [comment]
How big is your `mail.log`? Maybe it's just tons of data which makes this process take ages?

**Comment by lmfrazao** [comment]
yep, it's big, will log less stuff on it I guess.
I've removed the cronjob and it's ok, but no logs :)

**Comment by MrGeneration** [comment]
You could filter smtp related stuff into a independent log file if that helps to reduce the size, because by default modoboa will parse `mail.log` which by default also contains e.g. dovecot stuff. If not already configured, you may also want to rotate your logs - but that should be happening by default.

Edit: My fairly small setup has no issues with parsing. Could also be some log format issue during parsing, hard to tell if there's no errors.

**Comment by tonioo** [comment, marked as answer]
Parsing a big file can be pretty long so, as @MrGeneration suggests, you should rotate your log files and filter their content.
